etc/ci/upload_nightly.sh

```diff
@@ -16,25 +16,61 @@ usage() {
 
 
 upload() {
-    local nightly_filename nightly_timestamp
-    nightly_timestamp="$(date -u +"%Y-%m-%dT%H-%M-%SZ")"
-    nightly_filename="${nightly_timestamp}-$(basename "${2}")"
+    local nightly_filename
+    nightly_filename="${4}-$(basename "${2}")"
     local -r nightly_upload_dir="s3://servo-builds/nightly/${1}"
     local -r package_upload_path="${nightly_upload_dir}/${nightly_filename}"
     s3cmd --mime-type="application/octet-stream" \
           put "${2}" "${package_upload_path}"
     s3cmd cp "${package_upload_path}" "${nightly_upload_dir}/servo-latest.${3}"
 }
 
+update_brew() {
+  echo "Updating brew formula"
+
+  local package_url sha version script_dir tmp_dir nightly_filename
+
+  nightly_filename="${2}-$(basename "${1}")"
+  package_url="https://download.servo.org/nightly/macbrew/${nightly_filename}"
+  sha="$(shasum -a 256 ${1} | sed -e 's/ .*//')"
```

Comment:
`$()` creates a new quoting context, so you should be able to just do `"$(shasum -a 256 "${1}" | sed ...)"`